### ISSUE TITLE
[Feature request]change the order of alpha channel

### ISSUE THREAD

**Issue by tracyone** [IENT/YUView#146]
User can not change the order of alpha channel in rgb colorspace

ie. 

ARGB32 or RGBA 32

**Comment by ChristianFeldmann** [comment]
Yes. So far, we only support the alpha channel to be the last channel. So RGBA 32 is supported but ARGB 32 not.  It would be super helpful if you could provide an example rgb file so I can add this.

**Comment by tracyone** [comment]
Sorry, I will give you a raw rgb file tomorrow

**Comment by tracyone** [comment]
[rgb32.tar.gz](https://github.com/IENT/YUView/files/2420358/rgb32.tar.gz)

Sorry for the late reply.

Attach include two raw rgb file bgra 32 and argb 32

**Comment by tracyone** [comment]
![_026](https://user-images.githubusercontent.com/4246425/46089988-4e95a600-c1e2-11e8-926e-a01ab30d1d51.png)

**Comment by ChristianFeldmann** [comment]
I improved handling of RGB formats in the new_compression_file_reading branch and the alpha channel can now also be the first channel. There are the weirdest formats out there 😆 

If you want to, please try it out. If you are using windows, the compiled appveyor aritfacts are [here](https://ci.appveyor.com/project/aachenmax/yuview/build/1.0.158/artifacts).
